docs.mk

```diff
@@ -55,7 +55,7 @@ images: .mkdocs.dockerfile.timestamp
 		--sig-proxy=true \
 		--rm \
 		$(MKDOCS_IMAGE) \
-		/bin/bash -c "cd /d && $(MKDOCS) build; find /d/docs -exec chown $(UID):$(GID) {} \;"
+		/bin/bash -c "cd /d && $(MKDOCS) build && sed -i -e '/<replace-with-generated-content>/{r docs-src/spec.html' -e 'd}' docs/spec/index.html; find /d/docs -exec chown $(UID):$(GID) {} \;"
```

Comment:
(minor) might want to make it REPLACE_WITH_GENERATED_CONTENT to make it more clear what that string is doing

Reply:
Good call, that is better - updated.